core/tools.js

```diff
@@ -1129,21 +1131,12 @@
 		},
 
 		/**
-		 * Returns an array of passed object's keys.
-		 *
-		 *		console.log( CKEDITOR.tools.objectKeys( { foo: 1, bar: false } );
-		 *		// -> [ 'foo', 'bar' ]
-		 *
+		 * @inheritdoc CKEDITOR.tools.object#keys
 		 * @since 4.1
-		 * @param {Object} obj
-		 * @returns {Array} Object's keys.
+		 * @deprecated 4.12.0 Use {@link CKEDITOR.tools.object#keys} instead.
```

Comment:
Unfortunately, `@deprecated 4.12.0 Use {@link CKEDITOR.tools.object#keys} instead.` doesn't show the message in our docs, but I don't see any way to improve it:

![image](https://user-images.githubusercontent.com/1061942/59515450-68d1fa00-8ebf-11e9-9b66-f10e008f3093.png)

And the issue was already reported in Umberto.

Reply:
We could eventually add this information into method description itself, although it's smelling like duplication when Umberto will be fixed. Maybe let's just left as it is for now as we already have changelog entry for it?

Reply:
Let's leave it as is and wait for Umberto fix.